apps/nextjs-website/src/components/molecules/Chat/Chat.tsx

```diff
@@ -88,19 +88,23 @@ const Chat = ({
         ])
       ),
     ],
-    [queries, t, user, isFeedbackFormVisible]
+    [queries, t, user]
   ) satisfies Message[];
 
   const scrollRef = useRef<HTMLDivElement>(null);
+  const [queriesCount, setQueriesCount] = useState(0);
 
   useEffect(() => {
-    if (scrollRef.current) {
-      scrollRef.current.scrollIntoView({
-        behavior: instantScroll ? 'auto' : 'smooth',
-      });
+    if (messages.length != queriesCount) {
```

Comment:
Use strict equality (`!==`) instead of loose equality (`!=`)
```suggestion
    if (messages.length !== queriesCount) {
```